### ISSUE TITLE
Using waiter behind nginx and shinyproxy

### ISSUE THREAD

**Issue by deann88** [JohnCoene/waiter#42]
Hello,

I have written a shiny application, and it has a loading part where I am using the waiter.

The app runs perfectly when deployed locally with shinyproxy in a container (on localhost:8080), however, once deployed to a server behind a nginx server the loading screen never disappears and I do not get to see the app.

Nginx is used as a reverse proxy just to transfer traffic to the port 8080.

At the end of the ui file I have `waiter_show_on_load(html = spin_three_bounce())`, and at the end of the server file I have `waiter_hide()`. 

I have used `w <- Waiter$new(); w$show(); w$hide` in several tabs within the app, however I am not sure whether there is any issue with these uses, as I can not get through the initial loading screen - it just freezes there.

Any ideas what the issue might be?

Thanks

P.S. This issue is also posted [here](https://support.openanalytics.eu/t/using-waiter-with-shinyproxy/1514)

**Comment by JohnCoene** [comment]
Would you have reproducible example? It's difficult for me debug like this. I do not think nginx or docker/shinyproxy should be the issue here; it does not affect the JavaScript that runs in the browser.

Note: the [waiter demo](https://shiny.john-coene.com/waiter/) also runs on shiny server using nginx, so that cannot be the issue.

**Comment by deann88** [comment]
Sorry,

I was not able to provide a reproducible example as the issue indeed was with the nginx and shinyproxy settings.

In the application.yml file for shinyproxy I had to add:

> server:
> useForwardHeaders: true

And in the nginx config except for adding all recommended proxy setting as described [here](https://www.shinyproxy.io/security/), I had to add additionally this setting `proxy_buffering off;` in the `location` block.

So it was not a waiter issue at all. I have added a reply in the shinyproxy forum, so feel free to delete this issue.

Great job with this package, btw!
